app/controllers/courses/informed_consents_controller.rb

```diff
@@ -0,0 +1,98 @@
+# This controller is nested in the routing hierarchy as a child controller of 
+# courses_controller
+# see this tutorial for details on how we do this https://gist.github.com/jhjguxin/3074080
+
+module Courses
+  class InformedConsentsController < ApplicationController
+    layout 'courses'
+    before_action :load_parent
+    before_action :set_informed_consent, only: [:show, :edit, :update, :destroy]
+
+    load_and_authorize_resource :course
+    load_and_authorize_resource :informed_consent, through: :course
+
+    def index
+      @informed_consents = @parent.informed_consents.all
+
+      respond_to do |format|
+        format.html
+      end
+    end
+
+    def import
+      if params.has_key? :file
+        @parent.import_informed_consents(params[:file], params[:csv_header_map].split(','), params[:csv_header_toggle] == "true")
+        redirect_to course_informed_consents_path(@parent), notice: 'Informed Consent information imported.'
+      else
+        redirect_to course_informed_consents_path(@parent), alert: 'Please specify a file.'
+      end
+    end
+
+    
+    def show
+      @informed_consent = @parent.informed_consents.find(params[:id])
+    end
+
+    def new
+      @informed_consent = @parent.informed_consents.new
+    end
+
+    def edit
+    end
```

Comment:
Do we want an edit method AND an update method? It seems to me that they would largely overlap in responsibilities.